types/test/model.ts

```diff
@@ -98,6 +98,86 @@ UserModel.findCreateFind({
   }
 })
 
+/**
+ * Tests for findCreateOrCreate() type.
+ */
+class FindOrCreateModelTest extends Model {
+  static async testRowIsReturnedWhenAlreadyExists() {
+    let [model, created] = await this.findOrCreate({
+      fields: [ 'beta_user', 'username' ],
+      where: {
+        username: "test.user",
+      },
+      defaults: {
+        username: "test.user"
+      }
+    })
+
+    if (! created) {
```

Comment:
These tests are supposed to only testing syntax, not actually doing any db operations. I don't think these conditions or checks should be included in this code.

Just add some calls to `<Model>.findOrCreate` to assert correct typings or move these to `e2e` if you prefer that

Reply:
Ok, I have added only the typings because I have seen enough tests for `findOrCreate` in the integrations folder. Thank you for your guidance!